releases/note-source/v17_0.rst

```diff
@@ -0,0 +1,299 @@
+- ``base_Blendedness_raw_instFlux_parent`` was renamed to ``base_Blendedness_raw_parent_instFlux``;
+- ``base_Blendedness_abs_instFlux_child`` was renamed to ``base_Blendedness_abs_child_instFlux``;
+- ``base_Blendedness_abs_instFlux_parent`` was renamed to ``base_Blendedness_abs_parent_instFlux``;
+- ``deblend_psfFlux`` was renamed to ``deblend_psf_instFlux``.
+
+For more information, refer top `this community.lsst.org post`__ and :jira:`RFC-534`.
+
+__ https://community.lsst.org/t/change-in-naming-of-base-blendedness-and-meas-deblender-fields/3351
+
+.. _release-v17-0-task-metadata:
+
+Task metadata is now written as YAML
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously, metadata was stored as a mix of YAML__ and Boost.serialization__; now, only the former is used.
+For more information, refer to `this community.lsst.org post`__.
+
+__ https://yaml.org
+__ https://www.boost.org/doc/libs/1_69_0/libs/serialization/doc/index.html
+__ https://community.lsst.org/t/task-metadata-now-written-as-yaml/3192
+
+.. _release-v17-0-pybind11:
+
+Changes to wrapping C++ code into Python
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+DM uses pybind11__ to expose functionality written in C++ to the Python level.
+This release rethinks the way pybind11 is used, to increase robustness and reduce the size of compiled binaries.
+Although it is possible to mix the old and the new techniques within the same software stack, packages must be updated to the new system atomically.
+For more information, refer to `this community.lsst.org post`__ and the `pybind11 section in the Developer Guide`__.
+
+__ https://pybind11.readthedocs.io/en/stable/
+__ https://community.lsst.org/t/new-approach-and-tools-for-pybind11-wrappers/3291
+__ https://developer.lsst.io/pybind11/how-to.html
+
+.. _release-v17-0-astro-metadata-translator:
+
+Changes to metadata extraction from file headers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The new `astro_metadata_translator`__ package provides a unified interface for extraction of standardized information from image file headers.
+All camera support (“obs”) packages are being updated to use this new system, which will be required for the next generation of DM middleware.
+For more information, refer to `this community.lsst.org post`__.
+
+__ https://astro-metadata-translator.lsst.io
+__ https://community.lsst.org/t/changes-to-metadata-extraction-and-visitinfo-creation/3360
+
+.. _release-v17-0-logging:
+
+Revised logging system
+^^^^^^^^^^^^^^^^^^^^^^
+
+Log messages generated by the standard Python ``logging`` module during the execution of command-line tasks are now forwarded to the :ref:`lsst.log` system.
+Furthermore, utility functions have been added to make it easy to check that the expected log messages have been generated during testing.
+For more information, refer to `this community.lsst.org post`__.
+
+__ https://community.lsst.org/t/logging-changes/3580
+
+.. _release-v17-0-deprecations:
```

Comment:
I feel like we should probably mention the `obs_lsstSim` deprecation as well.

Reply:
Please add it to the branch is this is required.

Reply:
You want me to add text?

Reply:
If this is relevant for the release, I think the text should be added. You know probably better than me.